### ISSUE TITLE
[Feature Request]: Potential new hotkeys

### ISSUE THREAD

**Issue by RedDevilus** [PCSX2/pcsx2#5567]
### Description

Central location for requesting new hotkeys.

### Reason

Easier for tracking or for sharing more ideas.

### Examples

- [ ] Soft Reset (https://github.com/PCSX2/pcsx2/issues/4770) (Like the DS for Pokemon games where you do a combination)
- [ ] Turbo Hold (https://github.com/PCSX2/pcsx2/issues/5538) (instead of only a toggle)
- [ ] Swap-Disc/Booting/Browsing for a game
- [ ] Hardware Renderer swap (already has F9 for software renderer swap)
- [ ] Deinterlacing (to quickly see comparison and see what fits best)
- [ ] Blending accuracy (to quickly see comparison and see what fits best but probably basic for most)
- [ ] ReShade / Forum Shader or just shaders in general (used to be the Home button)
- [ ] Enabling cheats/patches (useful if you don't want/have Keyboard and mouse)
- [ ] Pause button (At the moment it's hardcoded to Escape)
- [ ] Debugger hotkeys (Perhaps less interesting but an idea nonetheless)
- [ ] OSD hotkeys
- [ ] Any other hotkey I can't come up with at the moment (Even for how nitpicky they could be)

**Comment by refractionpcsx2** [comment]
There's already a hotkey for Deinterlacing (F5), or am I missing something?

**Comment by RedDevilus** [comment]
Seemingly my brain. 😭

**Comment by DrEVILish** [comment]
Hotkey for Increase / Decrease internal resolution

**Comment by RedDevilus** [comment]
> Hotkey for Increase / Decrease internal resolution

![image](https://user-images.githubusercontent.com/24227051/159770914-c924ef1f-632d-461b-b718-42a472e6a5a1.png)
Already implemented and working.

**Comment by DrEVILish** [comment]
Awesome, please excuse me for being totally stupid: 1. which version is this in and 2: how do I do that?

**Comment by RedDevilus** [comment]
It's the upcoming Qt GUI. It's still a work in progress.

**Comment by Daisouji** [comment]
Volume: Inc/dec/mute

**Comment by TomTurbine** [comment]
Using a PS5 controller and wanting to bind the PlayStation logo to exit the emulator back to the front end I am setting up but no luck.

**Comment by Kuzma-Kuzmitch** [comment]
A hotkey for toggling the osd on/off  with the settings we already have enabled.
:)

**Comment by michaelx333** [comment]
Can we please have a hotkey for Reload Cheats/Patches button? I use it quite often to apply pnach changes and it's annoying having to navigate to menus all the time.

**Comment by DangerX7** [comment]
Yes, that would help a lot. I’ve made two tools for two different PS2 games, both of which rely primarily on PNACH files. I designed them to be simple and fast to use, but always looking for that reload option kind of kills the fun.

**Comment by kamfretoz** [comment]
> F7 TV shader with no F7 assigned

This has been added on #12954

**Comment by kamfretoz** [comment]
> Blending accuracy

Added in #14112 

> Enabling cheats/patches

not exactly enabling or disabling but i added patches reload hotkey on #14117
